### PR TITLE
Making meshnets a package.

### PR DESCRIPTION
**Pull request by AugustoPeres**
Addresses issue #44.

Previews:

Install
image

Import and python autocomplete
image

### CODE REVIEW

File: meshnets/modules/__init__.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+"""Init"""
+from .decoder import *
```
Comment (IvanPombo) [inline meshnets/modules/__init__.py:2]: Should we be more explicit?

**Review by IvanPombo — APPROVED** [review]
LGTM! Just left two minor comments.

File: meshnets/modules/__init__.py (hunk 1 above)
Comment (AugustoPeres) [reply meshnets/modules/__init__.py:2]: Only if there is something that we specifically do not want to be importable (I think)

Comment (fabiocruz) [reply meshnets/modules/__init__.py:2]: This isn't a good practice, since it omits what is being imported. Import all submodules you want instead.

**Review by fabiocruz — CHANGES_REQUESTED** [review]
Wildcard imports, `from <module> import *` should be avoided for clarity. See e.g. [this](https://peps.python.org/pep-0008/#imports).

**Comment by AugustoPeres** [comment]
> Wildcard imports, `from <module> import *` should be avoided for clarity. See e.g. [this](https://peps.python.org/pep-0008/#imports).

I was under the impression that this did not apply to `__init__` files. But I was wrong. Will change it now

File: meshnets/utils/__init__.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+                        GPUUsageMean, GradientNorm, GeometricBatchSize)
+from .data_loading import (load_edge_mesh_pv, edges_from_meshio_mesh,
+                           load_edge_mesh_meshio, load_triangle_mesh)
+from .data_processing import (edge_mesh_to_graph, triangle_mesh_to_graph,
+                              mesh_file_to_graph_data)
+from .data_visualization import (plot_mesh, plot_mesh_with_scalars,
+                                 plot_mesh_comparison)
+from .datasets import FromDiskGeometricDataset
+from .model_loading import load_model_from_mlflow
+from .model_training import train_model
```
Comment (fabiocruz) [inline meshnets/utils/__init__.py:13]: I think in this particular file you want to do e.g. `from . import callbacks`, as not to expose all utilities equally, e.g. `meshnets.utils.FromDiskGeometricDataset` and `meshnets.utils.GradientNorm`. I think it's useful to keep the submodule names, e.g. `meshnets.utils.datasets.FromDiskGeometricDataset` for context.

**Review by fabiocruz — APPROVED** [review]
Looking good, although I think you should import things slightly different in the `utils` submodule. See my comment below.

File: meshnets/utils/__init__.py (hunk 2 above)
Comment (AugustoPeres) [reply meshnets/utils/__init__.py:13]: Done
image

**Comment by AugustoPeres** [comment]
Going to merge: @VictorGillioz please take a look at this and issue #45 when you can

**Pull request merged**